qchem/setup.py

```diff
@@ -20,9 +20,7 @@
     "pennylane>=0.13",
     "scipy",
     "openfermion",
```

Comment:
Realized that recently @trbromley discovered that we'd need to pin `openfermion` (seems that's done in `requirements.txt`). Could consider `"openfermion>=0.10.0"` too (though that would diverge from `requirements.txt`.

The pinning is needed because with `openfermion==0.9.0` we're getting
```
/site-packages/openfermion/utils/_bch_expansion.py in <module>
     14 
     15 import itertools
---> 16 from scipy.misc import comb, factorial
     17 
     18 

ImportError: cannot import name 'comb' from 'scipy.misc' (/site-packages/scipy/misc/__init__.py)
```
```suggestion
    "openfermion==0.10.0",
```